packages/pwa-kit-extension-sdk/src/configs/webpack/override-stats-plugin.test.ts

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
Generally speaking this isn't following the pattern we have currently for testing webpack plugins. What we have now is more wholesome in that it tests the plugin in a real webpack compilation using a virtual file system. Did you think about using that pattern when writing these tests?

Reply:
Done. I updated the tests to use the virtual file system